### PR TITLE
Project Todo - Jessika Hage

### PR DESCRIPTION
**Pull request by jessika-hage**
Link to my deployed project:

https://jessikas-todos.netlify.app/

My reflections on how this week's project turned out:

See readme

### CODE REVIEW

File: code/src/components/TaskCard.js/DeleteButton.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          <DialogContent>Do you want to delete task?</DialogContent>
+        <DialogActions>
+          <DialogButton onClick={onDeleteDialog}>Cancel</DialogButton>
+          <DialogButton onClick={onClick}>Yes, delete!</DialogButton>
+        </DialogActions>
+        </DialogContainer>
+      </Dialog>
+    </>
+  )
+}
```
Comment (matildasvensson85) [inline code/src/components/TaskCard.js/DeleteButton.js:33]: Semimcolon missing

**Review by matildasvensson85 — APPROVED** [review]
Really nice project with a lot of nice features! Changing color scheme, adding categories to tasks, filtering tasks, etc. The styling is also very nice.

The code looks neat and well structured, Nothing to comment on, good job!

File: code/src/components/Header/ThemeButtons.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      <Tooltip title="Gray">
+        <GrayButton onClick={() => dispatch(themes.actions.changeTheme(grayTheme))}></GrayButton>
+      </Tooltip>
+      <Tooltip title="Black">
+        <BoldButton onClick={() => dispatch(themes.actions.changeTheme(boldTheme))}></BoldButton>
+      </Tooltip>
+      <Tooltip title="White">
+        <WhiteButton onClick={() => dispatch(themes.actions.changeTheme(whiteTheme))}></WhiteButton>
+      </Tooltip>
+    </>
```
Comment (KarolCislowski) [inline code/src/components/Header/ThemeButtons.js:59]: it is a lot of repeated code... Maybe there is a way to make it by using the `.map()` 🤓

File: code/src/components/TaskCard.js/DeleteButton.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import { Tooltip, Dialog, DialogActions, DialogContent } from "@material-ui/core";
```
Comment (KarolCislowski) [inline code/src/components/TaskCard.js/DeleteButton.js:2]: When you are importing more than 2 items from one library it is better to organize them in a column to make them easier to read and debug.

**Review by KarolCislowski — APPROVED** [review]
This is an awesome app 🦄 You built a very complicated logic the way it is nice and easy to read and understand! Grat Job!

**Pull request closed without merging**